frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/manyOfProperty.component.tsx

```diff
@@ -15,66 +15,38 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { TeamspacesHooksSelectors } from '@/v5/services/selectorsHooks/teamspacesSelectors.hooks';
 import { UsersHooksSelectors } from '@/v5/services/selectorsHooks/usersSelectors.hooks';
-import { FormMultiSelect } from '@controls/formMultiSelect/formMultiSelect.component';
 import { MultiSelectMenuItem } from '@controls/formMultiSelect/multiSelectMenuItem/multiSelectMenuItem.component';
-import { store } from '@/v4/modules/store';
-import _ from 'lodash';
+import { useSelector } from 'react-redux';
+import { selectJobs } from '@/v4/modules/jobs';
+import { FormMultiSelect } from '@controls/formMultiSelect/formMultiSelect.component';
 import { PropertyProps } from './properties.types';
 
-const JobsAndUsersProperty = ({
-	property: { name, readOnly, required },
-	defaultValue = [],
-	...props
-}: PropertyProps) => {
-	const teamspace = TeamspacesHooksSelectors.selectCurrentTeamspace();
-	const users = _.sortBy(UsersHooksSelectors.selectUsersByTeamspace(teamspace), 'firstName');
-	// TODO fix after "jobs" is accessible in v5
-	// @ts-ignore
-	const { jobs } = store.getState();
-	const jobItems = jobs.jobs.map(({ _id }) => (
-		<MultiSelectMenuItem key={_id} value={_id}>{_id}</MultiSelectMenuItem>
-	));
-	const userItems = users.map(({ user, firstName, lastName }) => (
-		<MultiSelectMenuItem key={user} value={user}>{`${firstName} ${lastName}`}</MultiSelectMenuItem>
-	));
+export const ManyOfProperty = ({ property, ...props }: PropertyProps) => {
+	const { name, readOnly, required, values } = property;
+	let items = [];
+
+	useSelector(selectJobs);
+
+	if (values === 'jobsAndUsers') {
+		const users = UsersHooksSelectors.selectCurrentTeamspaceUsers();
+		const jobs = useSelector(selectJobs);
+		items = users.map(({ user, firstName, lastName }) => ({ value: user, label: `${firstName} ${lastName}` }));
+		Array.prototype.push.apply(items, jobs.map(({ _id }) => ({ value: _id, label: _id })));
```

Comment:
If you are using `Array.prototype.push.apply` to add the jobs as single items, you can use the spread operator instead inside push like this
```ts
items.push(...jobs.map(({ _id }) => ({ value: _id, label: _id })))
```

Reply:
I'm avoiding spread operators in that line, otherwise I would have used
```
items = [...items, ...jobs.map(({ _id }) => ({ value: _id, label: _id }))];
````

Reply:
May I ask you why?